ADMINS.md

```diff
@@ -42,8 +43,20 @@ Act on [CODE_OF_CONDUCT](CODE_OF_CONDUCT.md) violations by revoking access, and
 
 ### Add/Remove Maintainers
 
-Perform administrative tasks, such as [adding](MAINTAINERS.md#adding-a-new-maintainer) and [removing maintainers](MAINTAINERS.md#removing-a-maintainer).
+Perform administrative tasks, such as [adding](RESPONSIBILITIES.md#adding-a-new-maintainer) and [removing maintainers](RESPONSIBILITIES.md#removing-a-maintainer).
+
+Please note that maintainers typically do not have admin-level permissions in their repos in this organization. Admin-level permissions allow for sensitive and destructive actions, such as managing security, or deleting a repository. Therefore, admin access in opensearch-project was designed to be deliberately centralized in ways that requires that two people to make any one sensitive change. If you need to perform an admin function, such as adding or removing a maintainer, please [follow the maintainer nomination process](RESPONSIBILITIES.md#becoming-a-maintainer), then ask to effect permissions by tagging `@admin` in your pull request, or the [#maintainers channel on the public Slack](https://opensearch.slack.com/archives/C05L60S4UBT). One of the above-mentioned admins will make this change for you.
 
 ### Adopt Organizational Best Practices
 
-Adopt organizational best practices, work in the open, and collaborate with other admins by opening issues before making process changes. Prefer consistency, and avoid diverging from practices in the opensearch-project organization.
\ No newline at end of file
+Adopt organizational best practices, work in the open, and collaborate with other admins by opening issues before making process changes. Prefer consistency, and avoid diverging from practices in the opensearch-project organization.
+
+## New Repos
+
+There are currently two ways new repositories may appear in the opensearch-project organization: creating a new repo and adopting, or moving a repo from outside of the organization into it. The process is the same.
+
+The AWS Open Source Program Office (OSPO) currently owns and manages the opensearch-project organization, and has permissions to create a new repo. While the admins above have admin-level permissions, they do not have permissions to create new repositories or move repositories into the organization.
+
+All new repositories inside opensearch-project follow the [security response process](SECURITY.md), and therefore require an Amazon team to be engaged when necessary. If you don't work for Amazon, and wish to create a repository in this organization, or move a repository into opensearch-project, please first discuss it in the [#maintainers channel on the public Slack](https://opensearch.slack.com/archives/C05L60S4UBT), and one of the above-mentioned admins will seek a team to support you. If you work for Amazon, and need a repo, you may need to take some additional steps. Please follow the instructions on [🔒 this internal Wiki](https://w.amazon.com/bin/view/OpenSearch_Project/GitHub_Repos/).
```

Comment:
Im not comfortable sharing internal wiki's. 
But anyway, an alternative I believe would be better is to have an issue template in this repo which describes "the request for creating/moving a repo". This would be even field for all requestor's (Community or Amazonian's) and obviously admins can follow up.

Reply:
What are your reasons for not being comfortable with sharing an internal link? This one is pretty vanilla IMO.

Reply:
Just exposing internal tools and their URLs. May be its just me :).

Reply:
I am not concerned about this internal wiki link, as that URL does not disclose any interesting information and its documented that its 'restricted' access.

Reply:
I think we all agree on a couple of things:

1. The link itself is not anything sensitive, and we must continue being careful about not disclosing any information that shouldn't be. 
2. It's totally ok for a company to ask its employees to follow an additional process. 

I like my idea of putting a link to a private wiki in open-source because it's "open-source first", and it has worked for me well in past life. I want employees at Amazon to be reading these open-source docs, then if they need to do something extra because the company asks them to, read something internal.

I would also, personally, be totally fine for another company with N+1 developers heavily involved in OpenSearch open-source (e.g. Aiven, cc: @reta) to put an internal link to their own wiki here.

I'm curious what @hyandell thinks.

Reply:
AWS managed open source projects should not include internal URLs. This includes wiki links.

Reply:
I removed it. But thought I'd try, I still think it's a harmless idea.